android/app/build.gradle

```diff
@@ -54,46 +54,14 @@ android {
         multiDexEnabled true
     }
 
-    def keystoreProperties = new Properties()
-    def keystorePropertiesFile = rootProject.file('key.properties')
-    if (keystorePropertiesFile.exists()) {
-        keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
-    }
-
-    signingConfigs {
-        release {
-            String filePath = keystoreProperties['storeFile']
-            if (filePath != null) {
-                keyAlias keystoreProperties['keyAlias']
-                keyPassword keystoreProperties['keyPassword']
-                storeFile file(filePath)
-                storePassword keystoreProperties['storePassword']
-            } else {
-                println("No storeFile provided, release builds are not possible")
-            }
-        }
-        debug {
-            String filePath = keystoreProperties['storeFileDebug']
-            if (filePath != null) {
-                keyAlias keystoreProperties['aliasDebug']
-                keyPassword keystoreProperties['passwordDebug']
-                storeFile file(filePath)
-                storePassword keystoreProperties['passwordDebug']
-            } else {
-                println("No storeFile provided, debug builds are using your local debug keystore")
-            }
-        }
-    }
-
     buildTypes {
         debug {
-            signingConfig signingConfigs.debug
-            applicationIdSuffix '.debug'
-            versionNameSuffix '-DEBUG'
             resValue "string", "app_name", "Breez Cloud - Debug"
         }
         release {
-            signingConfig signingConfigs.release
+            // TODO: Add your own signing config for the release build.
+            // Signing with the debug keys for now, so `flutter run --release` works.
+            signingConfig signingConfigs.debug
```

Comment:
Are the changes to this file part of the PR?

Reply:
They are intentional but was supposed to be on main branch. They revert  
- #623 

changes so we can keep building APK's via CI